### ISSUE TITLE
Wrong return number in types.h

### ISSUE THREAD

**Issue by firesurfer** [ros2/rcl#71]
I read some code for my C# wrapper and found these lines in the rcl types.h:

```c++
// rcl subscription specific ret codes in 4XX
#define RCL_RET_SUBSCRIPTION_INVALID 400
#define RCL_RET_SUBSCRIPTION_TAKE_FAILED 501
// rcl service client specific ret codes in 5XX
#define RCL_RET_CLIENT_INVALID 500
#define RCL_RET_CLIENT_TAKE_FAILED 501
```

Shouldn't be  `#define RCL_RET_SUBSCRIPTION_TAKE_FAILED 501` defined to 401 instead of 501 ?

**Comment by dirk-thomas** [comment]
That looks indeed like a copy-n-paste error. Can you please create a PR to address it?

**Comment by dirk-thomas** [comment]
Fixed by #72.
